lib/storage.js

```diff
@@ -54,8 +58,10 @@ async function getAsset(assetReference, directory, name, disableDownload) {
             // download http/https url into file
             const protocol = URL.parse(assetReference.url).protocol;
             if (protocol === DATA_PROTOCOL) {
+                console.log("creating asset from data url:", source.path);
                 await datauri.download(assetReference, source.path);
             } else {
+                console.log("downloading asset:", source.path);
```

Comment:
Aren't we supposed to redact asset paths and names to avoid leaking information?

Reply:
No, the _path_ does not include credentials. In the presigned URLs they are in the URL query parameters.

Reply:
But what if the customer names the asset one of their new product names that was not released yet and some how someone got a hold of the asset name and then leaked it to the public? 

Apple was very particular to this and that is why they hosted IPS/IS internally for unreleased Apple products so they could setup the pages they would use at release time.

Reply:
Logging here != leaked to the public.

We log this in many places and it is visible in metrics and logs/metrics on the AEM side to AEM devops. There is a limit to what we can hide without making it impossible to operate IMO.